src/convertToObject.js

```diff
@@ -6,7 +6,22 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
-}
+  const stylesArray = sourceString
+  .trim()
+  .split(';')
+  .map(style => style.trim())
+  .filter(style => style);
+
+  const stylesObj = {};
+
+  stylesArray.forEach(style => {
+    const [key, value] = style.split(':').map(part => part.trim());
 
+    if (key && value) {
+      stylesObj[key] = value;
+    }
```

Comment:
The use of `forEach` is correct as per the checklist, but consider checking if `style.includes(':')` before splitting to avoid creating key-value pairs from invalid strings.